feature-policy/resources/feature-policy-wakelock.html

```diff
@@ -0,0 +1,9 @@
+<script>
+'use strict';
+
+Promise.resolve().then(() => navigator.getWakeLock("screen")).then(wakelock => {
```

Comment:
I'd suggest changing this to an async function, with try/catch. Can we do anything useful with that error?

Reply:
I have one concern here, what is the error should be, "SecurityError"? But the spec does not explicitly state.

Reply:
Good catch! Should definitely add to the spec.